### PR TITLE
Del user story

### PR DESCRIPTION
**Pull request by codelawer**

### CODE REVIEW

File: client/src/apiClient.js [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
+            //route is specified in  server/api/index
+            method: 'GET',
+            headers: { Accept: 'application/json' },
+            mode: 'cors',
+        });
+        return response.json();
+    };
+
+    deleteMessage = async id => {
+        const response = await fetch(`${baseUrl}/api/message/delete/${id}`, {
```
Comment (herdani) [inline client/src/apiClient.js]: We don't need to have the `delete` word in the route, it' already stated with the http method

File: server/api/index.js [hunk 2]
```diff
@@ -68,6 +61,23 @@ apiRouter.post('/message/add', (req, res) => {
     });
 });
 
+// Delete Message
+apiRouter.delete('/message/delete/:id', (req, res) => {
+    const { id } = req.params;
+    const showAll = `select * from messages;`;
```
Comment (herdani) [inline server/api/index.js]: Not needed

File: server/api/index.js [hunk 3]
```diff
@@ -68,6 +61,23 @@ apiRouter.post('/message/add', (req, res) => {
     });
 });
 
+// Delete Message
+apiRouter.delete('/message/delete/:id', (req, res) => {
```
Comment (herdani) [inline server/api/index.js]: See previous comment: no delete word

File: server/api/index.js (hunk 2 above)
Comment (codelawer) [reply server/api/index.js]: showAll variable deleted and SQL query changed.

**Pull request merged**